metadata-ingestion/transformers.md

```diff
@@ -27,7 +27,7 @@ transformers:
 
 :::tip
 
-If you'd like to add more complex logic for assigning tags, you can use the more generic [`add_dataset_tags` transformer](./src/datahub/ingestion/transformer/add_dataset_tags.py), which calls a user-provided function to determine the tags for each dataset.
+If you'd like to add more complex logic for assigning tags, you can use the more generic add_dataset_tags transformer, which calls a user-provided function to determine the tags for each dataset.
```

Comment:
ideally add a link / callout to the right section of the docs

we should also emphasize that the `add_dataset_tags` transformer can be passed that method reference as an import string as well